### PR TITLE
feat(integrations): add support for pivotaltracker api

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Added configuration for pivotaltracker in providers.yaml
- Added pivotaltracker to docs-v2.
- Added pivotaltracker pages for Supported API groups in mint.json.
- Added pivotaltracker SVG file in template-logos folder.
- Consolidate pivotaltracker docs pages

## Issue ticket number and link
[EXT-55](https://linear.app/nango/issue/EXT-55/add-support-for-pivotaltracker-api)
## Test
This provider has been used successfully in local development to connect to pivotaltracker after authorizing using Nango.
The documentation update was reviewed using `npm run docs` and verifying on localhost.

### CODE REVIEW

File: docs-v2/integrations/all/pivotaltracker.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+-   [Generate an API token in your Pivotaltracker account](https://www.pivotaltracker.com/profile)
+-   [Pivotaltracker API docs](https://www.pivotaltracker.com/help/api)
+-   [Authentication and API token](https://www.pivotaltracker.com/help/api#Basics)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
+
+- Pivotaltracker uses `API_KEY` auth mode with ` X-TrackerToken: api_key` as a request header to access different endpoints.
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/pivotaltracker.mdx]:
```suggestion
- Pivotaltracker uses `API_KEY` auth mode with `X-TrackerToken: api_key` as a request header to access different endpoints.
```

File: docs-v2/integrations/productivity.mdx [hunk 2]
```diff
@@ -30,7 +30,7 @@ sidebarTitle: Productivity
     <Card title="Monday" href="/integrations/all/monday" color="#68a063" />
     <Card title="Mural" href="/integrations/all/mural" color="#68a063" />
     <Card title="Notion" href="/integrations/all/notion" color="#68a063" />
-    <Card title="Pingboard" href="/integrations/all/pingboard" color="#68a063" />
+    <Card title="Pivotaltracker" href="/integrations/all/pivotaltracker" color="#68a063" />
```
Comment (bodinsamuel) [inline docs-v2/integrations/productivity.mdx:36]: you removed pingboard

Comment (hassan254-prog) [reply docs-v2/integrations/productivity.mdx:36]: @bodinsamuel just updated my pull request. Thank you 🙏

**Review by bodinsamuel — APPROVED** [review]

**Review by khaliqgant — APPROVED** [review]

**Comment by khaliqgant** [comment]
@hassan254-prog can you resolve conflicts?

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**